HARSHAL/javascript/assignment15/assignment15Q1.js

```diff
@@ -0,0 +1,53 @@
+class BankDet {
+    constructor() {
+        this.acc_no = 0;
+        this.acc_name = "";
+        this.bal = 0;
+    }
+    get_data(acc_no, acc_name, bal) {
+        this.acc_no = acc_no;
+        this.acc_name = acc_name;
+        this.bal = bal;
+    }
+    deposit(d) {
+        return (this.bal = this.bal + d);
+    }
+    withdraw(w) {
+        console.log(this.bal);
+        try {
+            if (w > this.bal) throw alert("ERROR: INSUFFICIENT FUNDS");
+        } catch (err) {
+            return err;
+        }
+        if (w < this.bal) {
+            this.bal = this.bal - w;
+            try {
+                if (this.bal < 1000)
+                    throw alert("your balance is below 1000, Please deposit");
```

Comment:
using an alert with throw is not a good way, as the it will throw a number,text,object...read more about throw.
so it must be throw "Error:..........."